packages/block-library/src/navigation-link/link-ui.js

```diff
@@ -0,0 +1,202 @@
+/**
```

Comment:
Is it worth moving this out of `navigation-link`?

Reply:
What is `this`? Sorry the comment is on the first line of a comment.

Reply:
Sorry, I mean the whole file.